### PR TITLE
Refactor CSS files in src/screens/MemberDetail

### LINKED ISSUE #2503: Refactor CSS files in src/screens/CommunityProfile
# Refactor CSS files in src/screens/CommunityProfile

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### LINKED ISSUE #2511: Refactor CSS files in src/screens/MemberDetail
# Refactor CSS files in src/screens/MemberDetail

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by mandeepnh5**
**What kind of change does this PR introduce?**
This PR will Refactor CSS files in src/screens/MemberDetail 

**Issue Number:**
Fixes #2511 

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout and functionality for member detail views with new CSS classes.
	- Introduced responsive design elements and improved user interface components.

- **Bug Fixes**
	- Updated class names for consistency and specificity in styling, particularly for the date picker component.

- **Documentation**
	- Updated import paths for styles to reflect new structure.

### CODE REVIEW

**Comment by mandeepnh5** [comment]
@coderabbitai full review

**Comment by mandeepnh5** [comment]
@coderabbitai full review

File: src/style/app.module.css [hunk 1]
```diff
@@ -1564,6 +1563,546 @@ input[type='radio']:checked + label:hover {
...
+  .searchtitleMemberDetail {
+    margin-top: 30px;
+  }
+}
+
+/* User page */
+
+.memberfontcreatedbtn {
+  border-radius: 7px;
+  border-color: #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:1898]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: src/style/app.module.css (hunk 1 above)
Comment (mandeepnh5) [reply src/style/app.module.css]: ok done

**Review by palisadoes — APPROVED** [review]

**Pull request merged**